include/openenclave/host.h

```diff
@@ -65,6 +65,11 @@ OE_EXTERNC_BEGIN
  */
 #define OE_ENCLAVE_FLAG_SIMULATE 0x00000002u
 
+/**
+ *  Flag passed into oe_create_enclave to run the enclave with kss config.
+ */
+#define OE_ENCLAVE_FLAG_KSS 0x00000004u
```

Comment:
This flag is reserved for internal use. We do not allow oe_create_enclave() call to specifically set this KSS flag. 

Suggestion: Reword the comment and move it to line 78 (within @cond DEV) so DOXYGEN ignores it.

Reply:
Updated